README.md

```diff
@@ -109,7 +109,9 @@ A JSON histogramming configuration has the following parameters:
     * "det_range" (array of ints): the range of detectors to histogram (optional for hist1d)
     * "width" (int): the width of the detector (dethist only)
     * "height" (int): the height of the detector (dethist only)
-    * "num_bins" (int): the number of histogram bins (hist1d and hist2d only)
+    * "num_bins" int or array_like or [int, int]
```

Comment:
Let's pretend the option to pass only one bin size to the 2d doesn't exist.
We need to support it because NICOS uses it, but let's not advertise it ;)

Reply:
Done